Dockerfile

```diff
@@ -271,5 +271,13 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
 
 USER user
 
+FROM nitro-node-dev as nitro-node-split
+USER root
+
+RUN apt-get install -y xxd
+COPY scripts/split-val-entry.sh /usr/local/bin
+ENTRYPOINT [ "/usr/local/bin/split-val-entry.sh" ]
```

Comment:
I think this should not be default image, because it's only useful for some, takes a few more seconds to bring-up, might have weir issues if the user changes some of the parameters.. etc.